# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../linux-header/cyclades.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
diff -Naur linux-2.6.11.orig/include/linux/cyclades.h linux-2.6.11/include/linux/cyclades.h
--- linux-2.6.11.orig/include/linux/cyclades.h	2005-03-02 09:38:33.000000000 +0200
+++ linux-2.6.11/include/linux/cyclades.h	2005-03-07 17:52:56.716885160 +0200
@@ -67,6 +67,8 @@
 #ifndef _LINUX_CYCLADES_H
 #define _LINUX_CYCLADES_H
 
+#include <linux/compiler.h>
+
 struct cyclades_monitor {
         unsigned long           int_count;
         unsigned long           char_count;
